scylla/src/transport/session_builder.rs

```diff
@@ -472,6 +473,33 @@ impl SessionBuilder {
         self.config.fetch_schema_metadata = fetch;
         self
     }
+
+    /// Set the keepalive interval.
+    /// The default is `None`, it corresponds to no keepalive messages being send.
+    ///
+    /// # Example
+    /// ```
+    /// # use scylla::{Session, SessionBuilder};
+    /// # async fn example() -> Result<(), Box<dyn std::error::Error>> {
+    /// let session: Session = SessionBuilder::new()
+    ///     .known_node("127.0.0.1:9042")
+    ///     .keepalive_interval(std::time::Duration::from_secs(42))
+    ///     .build()
+    ///     .await?;
+    /// # Ok(())
+    /// # }
+    /// ```
+    pub fn keepalive_interval(mut self, interval: Duration) -> Self {
+        if interval <= Duration::from_secs(1) {
+            warn!(
+                "Setting keepalive interval to such low value ({:?}) is not recommended",
```

Comment:
Why is it not recommended? What does "such a low value" mean? Let's be explicit, e.g.

```
Setting the keepalive interval to low values is not recommended as it can have a negative impact on performance. Consider setting it above 1 second.
```